roles/kubernetes/secrets/files/make-ssl.sh

```diff
@@ -93,6 +93,8 @@ if [ -n "$MASTERS" ]; then
     gen_key_and_cert "kube-scheduler" "/CN=system:kube-scheduler"
     # kube-controller-manager
     gen_key_and_cert "kube-controller-manager" "/CN=system:kube-controller-manager"
+    # metrics aggregator
+    gen_key_and_cert "aggregator-proxy-client" "/CN=system:aggregator-proxy-client"
```

Comment:
This forces us to make that key any time we need to make keys... but that's fine.
This won't get generated automatically on existing installs. I think we need a follow up pr to handle this upgrade logic.

Reply:
If you change the {{ kube_api_requestheader_allowed_names }} now. Then it wont work. The CN has to match.

Should we insert the same here with the variable?

Reply:
Please change it to `"/CN=front-proxy-client"`

Reply:
And then change the {{ kube_api_requestheader_allowed_names }} likewise to front-proxy-client

?

Still, if you change the kube_api_requestheader_allowed_names in the defaults file or in k8s-cluster, then things will break, unless you change it directly in the make-ssl.ssh file ?